classroom.rb

```diff
@@ -0,0 +1,59 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student
+    student.classroom = self
+  end
+end
+
+class Student
+  attr_accessor :name, :classroom
+
+  def initialize(name)
+    @name = name
+    @classroom = nil
+  end
+end
+
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
+
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
+  end
+end
```

Comment:
- [ ] In the rental class Getters and Setters for @book and @person instance variables should be present
- [ ] I advise you to put each class in a separate file this will make your code base a lot easier to read.
- [ ] the rental should add itself to the person and the book using the `push` array method.

Reply:
done